scripts/aries_bdd_tests.sh

```diff
@@ -13,12 +14,17 @@ cd ./build/aries-framework-go || exit
 
 git checkout ${ARIES_FRAMEWORK_COMMIT}
 
-sed -i '' -e "1,/AGENT_REST_IMAGE.*/s/AGENT_REST_IMAGE.*/AGENT_REST_IMAGE=docker.pkg.github.com\/trustbloc\/agent-sdk\/agent-sdk-rest/" test/bdd/fixtures/agent-rest/.env
-sed -i '' -e "s/aries-agent-rest /agent-rest /" test/bdd/fixtures/agent-rest/docker-compose.yml
+if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
what if it's not macOS, but, let say, "linux"? Can you plz elaborate a bit more on this?

Reply:
For Mac `sed -i '' -e`, for linux `sed -i -e`. 
See https://unix.stackexchange.com/questions/13711/differences-between-sed-on-mac-osx-and-other-standard-sed#comment543105_131940